File: src/main/java/com/hank/guess/Game.java
package com.hank.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Random random = new Random();
        int secret = random.nextInt(10)+1;//(0~9)+1
        System.out.println(secret);
        int number = 0;
        while (number != secret){
            System.out.print("Please enter a number:");
            number = scanner.nextInt();
            if(number < secret){
                System.out.println("Higher");
            }else if(number > secret){
                System.out.println("Lower");
            }else{
                System.out.println("Great , The number is " + secret);
            }
        }

    /*int i = 0 ;
        do{
            System.out.println(i);
            i++;
        }while (i<3);*/
    }
}

File: src/main/kotlin/com/hank/Game.kt
package com.hank

import java.util.*

fun main() {
//    val scanner = Scanner(System.`in`)
    val secret = Random().nextInt(10)+1
    println(secret)
    var number = 0
    while (number != secret){
        print("Please enter a number:")
        number = readLine()!!.toInt()
//        number = scanner.nextInt()
        if(number > secret){
            println("Lower")
        }else if (number < secret){
            println("Higher")
        }else{
            println("Great,The number is $secret")
        }
    }


}